rel-eng/packages/server-postgresql-image

```diff
@@ -0,0 +1 @@
+5.1.1 containers/server-postgresql-image/
```

Comment:
let's start from `5.1.0`. the first time I'll tag and submit, this value will be automatically increased to `5.1.1`. If we start with 5.1.1 already,  the  first release will be on version `5.1.2`
```suggestion
5.1.0 containers/server-postgresql-image/
```